### ISSUE TITLE
installing the plugin in Eclipse 2022-12 brings unsolvable dependency issue

### ISSUE THREAD

**Issue by KRM1234** [lc-leuc/ABAPQuickFixS4Conversion#5]
Dear Lawrence Cabac.
The installation aborts and brings a dependency issue with missing osgi.bundle. Unfortunate it does not directly point to the missing bundle and OSGI seems to be a wide field with a lot of frameworks, that I do not use, or so far never needed. 
Therefore, my question, what specific bundle is needed. 
Details: 
Eclipse 2022-12 Java Developer Edition with SAP’s latest ADT ABAP Plugin 
Error Message:
Cannot complete the install because one or more required items could not be found.
  Software being installed: ABAP QuickFix S4C 1.4.1.202212211855 (de.leuc.adt.quickfix.feature.group 1.4.1.202212211855)
  Missing requirement: ABAP Quick Fix S4C Plugin 1.4.1.202212211855 (de.leuc.adt.quickfix 1.4.1.202212211855) requires 'osgi.bundle; com.abapblog.adt.quickfix 1.1.0' but it could not be found
  Cannot satisfy dependency:
    From: ABAP QuickFix S4C 1.4.1.202212211855 (de.leuc.adt.quickfix.feature.group 1.4.1.202212211855)
    To: org.eclipse.equinox.p2.iu; de.leuc.adt.quickfix [1.4.1.202212211855,1.4.1.202212211855]

Many Thanks in Advance.
KR Markus

**Comment by lc-leuc** [comment]
Dear Markus,

please install required current version of ABAP QuickFixes and try again.
Repository location has been changed recently. Please find it here: 
  https://fidley.github.io/ADTPluginsUpdateSite/UpdateSite/ 
or consult the project github page.

Best regards Lawrence

PS: Missing bundle is also shown in error message:
![image](https://user-images.githubusercontent.com/90344490/228533015-f1180144-6f8d-4703-8642-626ac8a93827.png)

**Comment by KRM1234** [comment]
Many thanks for the fast reply. 
I must admit, that I tried so search for it in the marketplace, but somehow was not lucky. 
For “osgi.bundle” I got too much results and for the “abapblog and “quickfix” I got no results. In the end I think it was the space between “Quick” and “Fix” that makes the difference. Small thing, big issue ;-).
Anyway, many thanks. That solved my problem, now everything is fine.
